Makefile

```diff
@@ -58,7 +58,10 @@ endif
 
 docker-test: install
 	go test -v -coverpkg="./..." -c -tags testruncli
-	PATH=$(BINPATH)/:$(PATH) APTLY_VERSION=$(VERSION) $(PYTHON) system/run.py --long $(TESTS) --coverage-dir $(COVERAGE_DIR) $(CAPTURE) $(TEST)
+	test -e aws.creds && . ./aws.creds; \
```

Comment:
Is this file automatically created during the `make docker-build-system-tests` (or any other dependant) target ?

Reply:
no, the file has to be place there manually, if it is there s3 tests will be enabled..

Reply:
Should we not indicate it in the ReadMe or in the project's wiki ?

Reply:
tthis is true, alongside with openstack swift and azure testing...

I will move it to a next PR, has nothing to do in here.

Reply:
Sounds great.

Let me know when you have moved the aws.creds part, and this PR will be Ok for me 👌🏻

Reply:
I moved it our of this PR